### DISCUSSION TITLE
k3s config files locations? on ubuntu 22

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [k3s-io/k3s#7127] [General]
could you please share location of all config files for k3s?
Is it the same for rancher?

**Comment by brandond** [comment]
https://docs.k3s.io/installation/configuration#configuration-file
You can also edit the systemd unit for k3s to add CLI flags, but the config file is preferred.

Rancher is installed to a Kubernetes cluster, not to nodes, so you won't find any configuration files on the node itself. This also isn't the right place to ask Rancher questions.
